src/test/test_imdb.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from scrape_up.imdb import Actor, BoxOffice, Celeb
+
+class IMDBTest(unittest.TestCase):
+    """
+    IMDB module test.\n
+    | Classes         | Details                              |
+    | --------------- | ------------------------------------ |
+    | `Actor`         | Tests related to the Actor class     |
+    | `BoxOffice`     | Tests related to the BoxOffice class |
+    | `Celeb`         | Tests related to the Celeb class     |
+    
+    """
```

Comment:
Can we add the function details instead of classes name that you tested? Similar to [this](https://github.com/Clueless-Community/scrape-up/blob/main/src/test/codechef_test.py#L10)

Reply:
Yes sure. Will be doing it.

Reply:
Hey @Master-Keshav, this issue is assigned to @aliat110. Do not work on this else there will be multiple PRs for same issue.
Try working on some other issues open.

Reply:
@nikhil25803 I guess I am assigned to this [issue](https://github.com/Clueless-Community/scrape-up/issues/816) and not this particular one.

Reply:
Neither of you are assigned to this issue.